content/docs/projects/trust-manager.md

```diff
@@ -56,79 +70,299 @@ spec:
         linkerd.io/inject: "enabled"
 ```
 
-Bundle currently supports the source types `configMap`, `secret` and `inLine`,
-and target type `configMap`.
+`Bundle` resources currently support several source types:
+
+- `configMap` - a `ConfigMap` resource in the trust-manager namespace
+- `secret` - a `Secret` resource in the trust-manager namespace
+- `inLine` - a manually specified string containing at least one certificate
+- `useDefaultCAs` - usually, a bundle of publicly trusted certificates
+
+The single currently supported target type is `ConfigMap`.
 
 #### Namespace Selector
 
-The target `namespaceSelector` can be used for scoping which Namespaces targets
-are synced to, supporting the field `matchLabels`. Please see
-[here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
-for more information and how label selectors are configured.
+A target's `namespaceSelector` is used to restrict which Namespaces your `Bundle`'s target
+should be synced to.
 
-If `namespaceSelector` is empty, a bundle target will be synced to all
-Namespaces.
+`namespaceSelector` supports the field `matchLabels`.
 
----
+Please see [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
+for more information about how label selectors can be configured.
+
+If `namespaceSelector` is empty, a `Bundle`'s target will be synced to all Namespaces.
+
+> ⚠️ A future update to trust-manager **will** change this behavior so that an empty namespace selector will sync only
+to the trust-manager namespace by default.
 
 ## Installation
 
-First, install [cert-manager](https://cert-manager.io/docs/installation/) to the
-cluster, and then the trust operator. It is advised to run the trust operator in
-the `cert-manager` namespace.
+### Demo
+
+If you've got Docker installed and you just want to play with trust-manager as soon as possible, we provide
+a `demo` command to get a [Kind cluster](https://kind.sigs.k8s.io/) set up with minimal fuss.
+
+```bash
+git clone --single-branch https://github.com/cert-manager/trust-manager trust-manager
+cd trust-manager
+make demo
+# kubeconfig is in ./bin/kubeconfig.yaml
+# kind cluster is called "trust"
+```
+
+The demo installation uses Helm, and roughly matches what you'd get by installing trust-manager into your own
+cluster using Helm - although it uses locally-built images rather than the ones we provide publicly.
```

Comment:
Do we want to explain this here or do we move this to the README.md file in the trust-manager project.

Reply:
I think there's value to having this here as sort of a "quick and easy way to experiment with trust-manager" - but agreed it's probs more useful in the README!

I raised https://github.com/cert-manager/trust-manager/pull/103

Reply:
Thx, I approved that PR.